Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/azure/serverless/spark/models/AnalyticsActivity.java

```diff
@@ -27,10 +27,12 @@
 import java.util.UUID;
 import java.util.Map;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 /**
  * The common Data Lake Analytics activity information.
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Maybe we should rebase the repo first since json ignore is added in the latest develop branch